gitpractice/javascript/task9.js

```diff
@@ -0,0 +1,49 @@
+//http://www.codewars.com/kata/prefill-an-array
+function prefill(number, v) {
+    var array = [];
+    if (number === 0 || number==="0")
+        return [];
+    else if (typeof number == "string") {
+        if (isNaN(parseInt(number)))
+            throw new TypeError(number + " is invalid");
+        if (isNaN(number))
+            throw new TypeError(number + " is invalid");
```

Comment:
you can put all this conditions in one single if, since you are throwing the same error for all the conditions